src/test/java/de/retest/recheck/ui/descriptors/RootElementFilterTest.java

```diff
@@ -0,0 +1,314 @@
+package de.retest.recheck.ui.descriptors;
+
+import static de.retest.recheck.ui.Path.fromString;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import de.retest.recheck.ignore.CompoundFilter;
+import de.retest.recheck.ignore.Filter;
+import de.retest.recheck.review.ignore.AttributeFilter;
+import de.retest.recheck.ui.Path;
+import de.retest.recheck.ui.image.Screenshot;
+import de.retest.recheck.ui.image.Screenshot.ImageType;
+
+class RootElementFilterTest {
```

Comment:
Did you copy some test code, especially `RootElementFilterTests` and below? Some parts contain a few legacy names etc., which we should clean up when we add new code.

Reply:
Yes, I did. I did not realize this at the time. I will clean this up.